chain/chunks/src/lib.rs

```diff
@@ -743,10 +751,20 @@ impl ShardsManager {
         if !checked_feature!("stable", ForwardChunkParts, protocol_version) {
```

Comment:
Good point! I think we needed to guard the original change by protocol upgrade because if some validators upgraded and some other don't, the ones that upgraded will wait for chunk parts in vain if the chunk part owner hasn't upgraded yet. Here we saved an upgrade because the change only affects chunk only producers and chunk only producers are not stabilized yet. Technically the problem I described can still affect shardnet and betanet, but I think with us owning these two networks and we can upgrade all of them at almost the same time, so it's fine to not do a protocol upgrade here.

Reply:
Ah and also, this is a more of a bug fix instead of feature, with the current code, the chunk only producers will always wait in vain. Even if only part of the network adopts this change, it's still better than before.